src/components/FoodList.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import FoodBox from './FoodBox'
+
+export default class FoodList extends Component {
+    render() {
```

Comment:
Since this component is only running a map, you could try to implement the map directly in App.js in a couple of lines, as to avoid unnecessary code or components.